src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSidebar.tsx

```diff
@@ -65,30 +65,32 @@ export const PanelSideBar = <TPanelItemId extends string, TPanelItem>(props: Pan
     };
 
     const PanelItemsRenderer = (props: { items: PanelItem<TPanelItemId, TPanelItem>[] }) => {
+      // eslint-disable-next-line react/prop-types
       const { items } = props;
-      return items
-        ?.filter((x) => !hiddenMenuItemIds.includes(x.id))
-        .map((item, index) =>
-          renderTilesAsLinks ? (
-            <LinkRenderer key={index} item={item}>
-              <ButtonIcon item={item} />
-            </LinkRenderer>
-          ) : (
-            <ButtonIcon key={index} item={item} />
-          ),
-        );
+      return (
+        items
+          // eslint-disable-next-line react/prop-types
+          ?.filter((x) => !hiddenMenuItemIds.includes(x.id))
+          .map((item, index) =>
+            renderTilesAsLinks ? (
+              <LinkRenderer key={index} item={item}>
+                <ButtonIcon item={item} />
+              </LinkRenderer>
+            ) : (
+              <ButtonIcon key={index} item={item} />
+            ),
+          )
+      );
     };
```

Comment:
not really the concern of your PR but maybe since you are working on it anyways:

Could you move this component outside? Components inside components are bad practice:
https://www.developerway.com/posts/react-re-renders-guide#part3

Could you please check if this rule would detect it? 
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

And if so please enable it. 

Thanks 🙏

Reply:
moved 🙏

Reply:
and yes it would/will detect it (is now already added to the package and merged)